lambda/waf-to-splunk-kinesis-data-transformation/index.js

```diff
@@ -27,6 +31,6 @@ exports.handler = async (event, context) => {
             data: Buffer.from(JSON.stringify(data), 'utf8').toString('base64')
         }
     });
-    console.log(`Processing completed. Successful records ${output.length}.`)
+    console.log(`Processing completed. Successful records ${output.length}. Dropped ${droppedCount}. Ok ${okCount}.`)
```

Comment:
I like you've added the `dropped` and `ok` counts 👍 Maybe change `Successful` to `Total` since it's a bit confused with the new `Ok` total?

Reply:
All records are returned whether they're dropped or not. I'll update it be clearer.